source/AndroidResolver/src/SettingsDialog.cs

```diff
@@ -502,6 +512,27 @@ private class Settings {
                                 settings.localMavenRepoDir)));
             }
 
+            if (settings.patchMainTemplateGradle) {
+                GUILayout.BeginHorizontal();
+                GUILayout.Label("Use relative repository path", EditorStyles.boldLabel);
+                settings.useRelativeRepoPath =
+                    EditorGUILayout.Toggle(settings.useRelativeRepoPath);
+                GUILayout.EndHorizontal();
+
+                if (settings.useRelativeRepoPath) {
+                    GUILayout.Label(
+                        "The mainTemplate.gradle file will be patched with a relative path to " +
+                        "the Local Maven Repository. This can be used to limit file changes when " +
```

Comment:
Seems like this new flag only matters when GradleProjectExportEnabled is true. 

Perhaps change to something like "The mainTemplate.gradle file will be patched with a relative path to the Local Maven Repository for an exported Android project."

Reply:
That is correct. I've updated the description to match your suggestion.